### ISSUE TITLE
macOS installer option to not add to Dock and forcefully restart it

### ISSUE THREAD

**Issue by Allendar** [input-output-hk/daedalus#1895]
This might sound like a trivial thing, but for a macOS user to give their password to an installer, it expects that the installer does nothing else than install the app to the Apps folder and maybe generate some basic Library (core/user) patches.

The Windows-esque way of forcing the App onto the Dock and rebooting it (causing all windows to be forced forward onto the desktop) is not a desired effect for most macOS users.

Is it maybe possible to either add a checkbox in the installer's final step (not sure if Electron has some stuff out of the box for this?) that we can uncheck so the add-to-Dock and forceful restart of it doesn't occur?

Thanks in advance and keep up the amazing work!

**Comment by mareksoha** [comment]
yep, this is very very very annoying "feature". It still persist in newest versions even I was asking support to do something with it (This is a follow-up to your previous request #99720 "Daedalus - keep in dock in).
Can we please bump it up a little bit?

**Comment by mareksoha** [comment]
any update on this please? it should be easy change though.

**Comment by disassembler** [comment]
Closing as part of issue backlog cleanup — issues older than 90 days are being retired to give the new team a clean slate. Please open a new issue if this is still relevant.
